charts/kueue-ks/values.yaml

```diff
@@ -0,0 +1,68 @@
+# Default values for kueue-ks.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
```

Comment:
I assume this is not used in the chart, right ? I would suggest to set  it to empty. Even better, remove all that is not used in the chart (probably most of it)